orchagent/pfcactionhandler.cpp

```diff
@@ -441,7 +441,7 @@ PfcWdZeroBufferHandler::PfcWdZeroBufferHandler(sai_object_id_t port,
         return;
     }
 
-    setPriorityGroupAndQueueLockFlag(portInstance, true);
```

Comment:
PfcWdZeroBufferHandler is also used by vendors other than MLNX. https://github.com/Azure/sonic-swss/blob/201911/orchagent/orchdaemon.cpp#L320 & https://github.com/Azure/sonic-swss/blob/201911/orchagent/orchdaemon.cpp#L363. Are we planning to include this change for other vendors a well.

Reply:
Yes